src/OrderFormAcceptanceTests.Steps/Steps/CatalogueSolutions.cs

```diff
@@ -482,7 +482,7 @@ public void GivenFillsInTheCatalogueSolutionEditFormWithValidData(string quantit
                 Test.Pages.OrderForm.ClickRadioButton(1);
             }
 
-            int.TryParse(Test.Pages.OrderForm.GetPriceInputValue(), out int maxValue);
+            _ = int.TryParse(Test.Pages.OrderForm.GetPriceInputValue(), out int maxValue);
```

Comment:
If `GetPriceInputValue()` is out of bounds, `maxValue` will be `0` is that the desired behaviour? Would it be better to use `int.Parse` and allow it to throw in the case of programmer error?